theme_avantgarde/static/src/scss/primary_variables.scss

```diff
@@ -164,25 +176,24 @@ $o-theme-font-configs: (
 
 $o-website-values-palettes: (
     (
-        'color-palettes-name': 'avantgarde-1',
-
-        'font-size-base': (14 / 16) * 1rem,
-        'font': 'Heebo',
-        'headings-font': 'Playfair Display',
-        'navbar-font': 'Playfair Display',
-        'buttons-font': 'Playfair Display',
-
-        'layout': 'boxed',
-        'header-template': 'magazine',
-        'hamburger-type': 'off-canvas',
+        'color-palettes-name': 'avantgarde-3',
+        'layout': 'full',
+
+        'font-size-base': 1rem,
+        'font': 'Syne',
+        'headings-font': 'Syne',
+        'navbar-font': 'Syne',
+        'buttons-font': 'Syne',
+
+        'header-template': 'hamburger',
+        'hamburger-type': 'default',
+        'hamburger-position': 'right',
         'menu-box-shadow': false,
```

Comment:
Apparently, it's no use setting it to `false`, since it is set to `null` by default, so it is not working in this theme.

I've seen that a similar declaration of `'menu-box-shadow': false` in Paptic, Cobalt and Bewise, so I thought it would be working here too.

Reply:
> I've seen that a similar declaration of `'menu-box-shadow': false` in Paptic, Cobalt and Bewise

Yes, that's true. We should fix it in these themes.  👍